client/systems_test.go

```diff
@@ -324,21 +324,46 @@ func (cs *clientSuite) TestRequestSystemInstallEmptySystemLabel(c *check.C) {
 	c.Check(cs.req, check.IsNil)
 }
 
-func (cs *clientSuite) TestRequestSystemInstallHappy(c *check.C) {
+func (cs *clientSuite) TestRequestSystemInstallShadowsExistingStruct(c *check.C) {
```

Comment:
I think the test should have kept the previous name, there is anyway a comment inside about the shadowing issue